Src/Common/CrossComponentCorrelationEventSource.cs

```diff
@@ -15,12 +15,12 @@
     /// <summary>
     /// ETW EventSource tracing class.
     /// </summary>
-    [EventSource(Name = "Microsoft-ApplicationInsights-Extensibility-CrossComponentCorrelation")]
-    internal sealed class CrossComponentCorrelationEventSource : EventSource
+    [EventSource(Name = "Microsoft-ApplicationInsights-Extensibility-AppMapCorrelation")]
+    internal sealed class AppMapCorrelationEventSource : EventSource
```

Comment:
Two of them:
https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/75373e57dcf8d646c54ee188461c373f2cc98939/Test/PerformanceCollector/FunctionalTests/Helpers/EtwEventSession.cs#L11

https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/de8b7c3a4595ae4de6625814a54462b16663d004/Test/Web/FunctionalTests/FunctionalTests/Helpers/EtwEventSession.cs#L11